ledger-service/http-json/src/main/scala/com/digitalasset/http/dbbackend/ContractDao.scala

```diff
@@ -266,3 +304,46 @@ object ContractDao {
     val SqlState = "STALE_OFFSET_EXCEPTION"
   }
 }
+
+/*
+  TODO below values are hardcoded for now, refactor to be picked up as cli flags/ props later.
```

Comment:
does this comment still apply given that now it’s configurable?

Reply:
nope will cleanup